Source/Tests/FEXLoader.cpp

```diff
@@ -396,6 +377,15 @@ int main(int argc, char **argv, char **const envp) {
   auto SyscallHandler = Loader.Is64BitMode() ? FEX::HLE::x64::CreateHandler(CTX, SignalDelegation.get())
                                              : FEX::HLE::x32::CreateHandler(CTX, SignalDelegation.get(), std::move(Allocator));
 
+  auto Mapper = [&SyscallHandler](auto && ...args){ return SyscallHandler->GuestMmap(args...); };
```

Comment:
Nifty! I think ubuntu 20.04 that we have as a target though maxes out at clang11. I'll check if it compiles tomorrow.

Reply:
Oh, remembered why I didn't use that one, it's because ::GuestMmap are dynamic and the syscall handler we have is a base class pointer. Though, moving them to internal virtuals to share the code will solve that.

Reply:
Tested with `clang version 10.0.0-4ubuntu1` (shipped ubuntu 20.04-3) and `std::bind_front` exists and works for the non-virtual, and virtual case.

In the case of `&BaseClass::virtualFn` the implementation defined in the subclass is called via runtime resolution through the vtable, though I'm not sure I want to know what kind of asm that generates.

Reply:
Updated to use `bind_front`